examples/pennylane_optimization_in_pennylane.py

```diff
@@ -0,0 +1,92 @@
+r"""
+.. _optimization_in_pennylane:
+
+Optimization in PennyLane
+=========================
+
+Now that we are familiar with the different operations that are
+available and how quantum gradients are calculated in PennyLane, we can
+carry out cost/objective optimization.
+
+The gradient descent algorithm has two steps:
+
+1. Compute the gradient of cost function: :math:`\nabla_\alpha C`
+2. Update the parameters :math:`\alpha` in proportion to this gradient:
+
+   .. math:: \alpha \mapsto \alpha - \eta \nabla_\alpha C
+
+The scaling factor :math:`\eta` is known as the *learning rate*.
+
+   This procedure is carried out automatically by the PennyLane
+   :class:`GradientDescentOptimizer <pennylane.optimize.GradientDescentOptimizer>`
+   class object. This function needs to be called with one argument: the
+   stepsize for the gradient descent algorithm.
```

Comment:
```suggestion
   initial stepsize for the gradient descent algorithm.
```

Reply:
`initial stepsize`? Are we considering that the stepsize can change during the gradient descent process? Does PennyLane have that feature?